### PR TITLE
Script collecting run time of benchmark policy

### PR DESCRIPTION
**Pull request by larsewi**
This script may be useful for others. Merge at will.

Ticket: None
Changelog: None
Signed-off-by: larsewi <lars.erik.wik@northern.tech>

### CODE REVIEW

**Comment by olehermanse** [comment]
Tagging some "shell-gurus" for review ;)

File: docs/benchmark.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+done
+
+# Sort array of results
+IFS=$'\n' sorted=($(sort <<<"${array[*]}")); unset IFS
+
+echo
+echo "*** RESULTS ***"
+
+# Max time
+max=$(echo "$NUM_RUNS - 1" | bc -l)
```
Comment (nickanderson) [inline docs/benchmark.sh:30]: Maybe avoid reliance on `bc`?

```
max=$(($NUM_RUNS-1))
```

**Comment by nickanderson** [comment]
perhaps overkill for this, but you should know about https://argbash.io/

File: docs/benchmark.sh (hunk 1 above)
Comment (larsewi) [reply docs/benchmark.sh]: I was only able to do this on integers, not reals. Am I doing something wrong?

**Comment by larsewi** [comment]
Thank you for showing me argbash @nickanderson !

**Review by vpodzime — COMMENTED** [review]
My :eyes:!!! Have you ever heard about Perl? I mean, seriously. It will also be available on more systems than *bc* these days.

**Comment by nickanderson** [comment]
> My !!! Have you ever heard about Perl? I mean, seriously. It will also be available on more systems than _bc_ these days.

https://github.com/clarity20/shellmath

**Review by craigcomstock — APPROVED** [review]

**Pull request merged**